### DISCUSSION TITLE
Return policy behavior

### DISCUSSION THREAD

**Discussion by assaflei** [etsy/open-api#856] [Q&A]
Following the [latest return policy changes](https://github.com/etsy/open-api/discussions/761), I started getting the `return policy cannot be null` messages as part of a process of changing listings from active to inactive ([updateListing method](https://developer.etsy.com/documentation/reference/#operation/updateListing))

I noticed a strange behavior.
The listing state does change to `edit` but additional change to the description field is not applied

Is this an expected behavior?

**Comment by PMStanley** [comment]
I ask what actions you're taking here?

Are you pushing an update via "updateListing" that sets the 'state' to 'inactive' and you're also updating 'description' at the same time?  Do the listings this happen with already 'return_policy_id' set to a value other than null?

**Reply by assaflei** [reply]
> Are you pushing an update via "updateListing" that sets the 'state' to 'inactive' and you're also updating 'description' at the same time

Yes

> Do the listings this happen with already 'return_policy_id' set to a value other than null?

No, I see the message returned when the value is null
